.github/workflows/python-deps.yml

```diff
@@ -28,9 +28,11 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Install prerequisites
-        # brew can be finicky but it does not always means that the rest of the job will fail
+        env:
+          HOMEBREW_NO_AUTO_UPDATE: on
```

Comment:
The `python-deps` workflow is not intended to test against user's dev env setup right?

Reply:
No. We're using `brew` because it's available and Mac and Linux.

No auto update with just `brew install` is to make it execute faster.

If we had kept `make prerequisites` which calls `brew bundle`, it might have saved updating to some new library that breaks as we see on CI

From the man page:
```
HOMEBREW_NO_AUTO_UPDATE
              If set, do not automatically update before running some commands e.g. brew install, brew upgrade and brew tap.
```